content/blog/2018/egress-mongo/index.md

```diff
@@ -214,6 +217,8 @@ instructions in this section. Alternatively, if you do want to direct your traff
     Also note that when the protocol `TCP` is specified, the configuration is not specific for MongoDB, but is the same
     for any other database with the protocol on top of TCP.
 
+    Note that the host of your MongoDB is not used in TCP routing, so you can use any host, for example `my-mongo-fake-host.com`. Notice the `STATIC` resolution and the endpoint with the IP of your MongoDB service. Once you define such an endpoint, you can access MongoDB services that do not have an FQDN hostname.
```

Comment:
I think the name `my-mongo-fake-host.com` seems like a hack. Although the host isn't used for runtime routing, it is used for config matching (gateway and destination rules hosts need to match), so I think we should come up with a better recommended naming convention to use for TCP service entries like this. We should document a convention somewhere, and use it in this blog IMO.

Maybe something like `my-mongo.tcpsvc` ?

Reply:
@frankbu `my-mongo.tcpsvc` sounds good, I will use it.